src/components/Menu.vue

```diff
@@ -51,7 +51,11 @@ export default {
         this.sortBy
       );
 
-      return data;
+      if (Array.isArray(data)) {
```

Comment:
in which case can this happen, since the default value is `[]`

Reply:
This can happen when `getFacetValues` returns `{data: null}` like in the tests.